src/platform/silabs/efr32/OTAImageProcessorImpl.cpp

```diff
@@ -122,16 +128,16 @@ void OTAImageProcessorImpl::HandlePrepareDownload(intptr_t context)
 
     if (imageProcessor == nullptr)
     {
-        ChipLogError(SoftwareUpdate, "ImageProcessor context is null");
+        ChipLogError(SoftwareUpdate, "%s: imageProcessor context is null", __func__);
         return;
     }
     else if (imageProcessor->mDownloader == nullptr)
     {
-        ChipLogError(SoftwareUpdate, "mDownloader is null");
+        ChipLogError(SoftwareUpdate, "%s: mDownloader is null", __func__);
         return;
     }
 
-    ChipLogProgress(SoftwareUpdate, "HandlePrepareDownload");
+    ChipLogProgress(SoftwareUpdate, "%s: started", __func__);
```

Comment:
@rosahay-silabs adding the `__func___` is bad practice to have in a stack. this should only be done for debug purposes. It causes bloats for nothing. SHould be removed

Reply:
Added them to quick resolve where error logs would be coming from in case of an error, since the logs looked to be duplicated in multiple locations so an unique identifier was required.